code/game/machinery/teleporter.dm

```diff
@@ -196,7 +196,7 @@
 	name = "teleporter horizon generator"
 	desc = "This generates the portal through which you step through to teleport elsewhere."
 	icon_state = "tele0"
-	var/accurate = 0
+	var/accurate = 0 //Accuracy is no longer used.
```

Comment:
Why not just delete the variable then?

Reply:
I'd like to leave the code because it might help people with multi tile machines in the future. I know a few other ones exist but I feel like its valuable.
I can comment the var though.